com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Adjustment/ContrastNode.cs

```diff
@@ -1,7 +1,7 @@
 using System.Collections.Generic;
-using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
 
-namespace com.unity.shadergraph.defs
+namespace UnityEditor.ShaderGraph.Defs
```

Comment:
Is there a case to have definitions live in a separate namespace?

Reply:
Yes. When we spoke about the `FunctionDescriptor` concept, it seemed appropriate to separate it from the rest of the API. If it becomes a core ShaderGraph concept, we might want to move it into a shared namespace.

Secondarily, `Defs` namespace is problematic because it contains things like `Vector` and `Matrix`.